js/client-battle.js

```diff
@@ -1332,6 +1332,19 @@
 			var basePower = move.basePower;
 			var basePowerComment = '';
 			var thereIsWeather = (this.battle.weather in {'sunnyday': 1, 'desolateland': 1, 'raindance': 1, 'primordialsea': 1, 'sandstorm': 1, 'hail':1});
+			if (myPokemon.item) {
+				var item = Tools.getItem(myPokemon.item).name;
+				var pos = this.choice.choices.length;
+				var moveType = this.getMoveType(move, this.battle.mySide.active[pos]);
+				var itemInfo = item.split(' ');
+				var name = move.name;
+				if (itemInfo[1] == 'Gem' && moveType == itemInfo[0] && basePower && (name != 'Struggle' || name != 'Water Pledge' || name != 'Grass Pledge' || name != 'Fire Pledge' || name != 'Fling')) {
+					basePower *= this.battle.gen >= 6 ? 1.3 : 1.5;
+					var temp = item + ')';
+					var comment = ' (Boosted by ';
+					basePowerComment = comment + temp;
```

Comment:
This should probably just all be one line:

    basePowerComment += ' (Boosted by ' + item + ')';